src/models/account.cpp

```diff
@@ -507,6 +526,185 @@ int Account::importFromCSV(const std::string& path)
     return imported;
 }
 
+int Account::importFromOFX(const std::string& path)
+{
+    int imported{ 0 };
+    unsigned int nextId{ getNextAvailableTransactionId() };
+    const std::regex xmlTagRegex("^<([^/>]+|/STMTTRN)>(?:[+-])?([^<]+)");
+    std::smatch xmlTagMatch;
+    Transaction *transaction{ nullptr };
+    bool skipTransaction{ false };
+
+    std::ifstream file{ path };
+    if (file.is_open())
+    {
+        std::string line;
+        while (getline(file, line))
+        {
+            // Extract tag and content
+            if (std::regex_match(line, xmlTagMatch, xmlTagRegex))
+            {
+                const std::string& tag{ xmlTagMatch[1].str() };
```

Comment:
std::smatch::str() does not return a const reference. Therefore, the variable type should be `std::string` and not `const std::string&`